packet.go

```diff
@@ -734,63 +736,63 @@ func (p *sshFxpFsetstatPacket) UnmarshalBinary(b []byte) error {
 }
 
 type sshFxpHandlePacket struct {
-	Id     uint32
+	ID     uint32
 	Handle string
 }
 
 func (p sshFxpHandlePacket) MarshalBinary() ([]byte, error) {
 	b := []byte{ssh_FXP_HANDLE}
-	b = marshalUint32(b, p.Id)
+	b = marshalUint32(b, p.ID)
 	b = marshalString(b, p.Handle)
 	return b, nil
 }
 
 type sshFxpStatusPacket struct {
-	Id uint32
+	ID uint32
 	StatusError
 }
 
 func (p sshFxpStatusPacket) MarshalBinary() ([]byte, error) {
 	b := []byte{ssh_FXP_STATUS}
-	b = marshalUint32(b, p.Id)
+	b = marshalUint32(b, p.ID)
 	b = marshalStatus(b, p.StatusError)
 	return b, nil
 }
 
 type sshFxpDataPacket struct {
-	Id     uint32
+	ID     uint32
 	Length uint32
 	Data   []byte
 }
 
 func (p sshFxpDataPacket) MarshalBinary() ([]byte, error) {
 	b := []byte{ssh_FXP_DATA}
-	b = marshalUint32(b, p.Id)
+	b = marshalUint32(b, p.ID)
 	b = marshalUint32(b, p.Length)
 	b = append(b, p.Data[:p.Length]...)
 	return b, nil
 }
 
 func (p *sshFxpDataPacket) UnmarshalBinary(b []byte) (err error) {
-	if p.Id, b, err = unmarshalUint32Safe(b); err != nil {
+	if p.ID, b, err = unmarshalUint32Safe(b); err != nil {
```

Comment:
please delete the named return argument and add
```
var err error
```
at the top.

my opinion is named return arguments should only be used with naked returns, and if there are no naked returns, there should be no named return arguments.